Example/Tests/Tests.m

```diff
@@ -79,7 +79,11 @@ - (SCORStreamingAnalytics *)create;
             anonymousId:nil
             traits:@{ @"name" : @"Milhouse Van Houten",
                       @"gender" : @"male",
-                      @"emotion" : @"nerdy" }
+                      @"emotion" : @"nerdy",
+                      @"isMarried" : @YES,
+                      @"kids" : @[@"Dennis", @"Donald", @"Bunny"],
+                      @"age": @30
```

Comment:
Milhouse is 10 and has no kids.  :p

Reply:
Otherwise, looks good to me.  Default Objective C stringification of objects and related tests.